proto/server/PI/proto/pi_server.h

```diff
@@ -17,6 +17,9 @@
  * Antonin Bas (antonin@barefootnetworks.com)
  *
  */
+#include <PI/frontends/proto/device_mgr.h>
```

Comment:
`PI/frontends/proto/device_mgr.h` is a C++ header, you cannot include it in this header which is meant to be usable by both C++ and C compilers (which is why as you can see, there is no `PIGrpcServer` class).
Since you are only including it for the definition of `device_id_t` and nothing else, I would not bother and simply hardcode it to 64-bit.

Reply:
Fixed